pkg/helpers/queuekey.go

```diff
@@ -37,7 +37,7 @@ const (
 
 	RegistrationWebhookSecret  = "registration-webhook-serving-cert"
 	RegistrationWebhookService = "cluster-manager-registration-webhook"
-	WorkWebhookSecret          = "work-webhook-serving-cert"
+	WorkWebhookSecret          = "work-webhook-serving-cert" // #nosec G101
```

Comment:
what does this nosec G101 mean?

Reply:
G101 means gosec think it maybe a Hardcoded credentials problem:https://securego.io/docs/rules/g101.html

We add this comment to tell gosec this is a false positive alarm when it scanning the code.